.travis.yml

```diff
@@ -15,12 +13,16 @@ matrix:
         # Lint the first commit in the PR.
         - \[ -z "$TRAVIS_COMMIT_RANGE" \] || (echo -e '\nLinting the commit message according to the guidelines at https://goo.gl/p2fr5Q\n' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata)
     - name: "Test Suite"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-4.9
       install:
-        - ./configure
-        - make -j2 V=
+        - CC='ccache gcc-4.9' && CXX='ccache g++-4.9' python configure.py --verbose
+        - CC='ccache gcc-4.9' && CXX='ccache g++-4.9' make -j2 V=
       script:
-        - make -j2 test
+        - make test
```

Comment:
Refs: https://github.com/nodejs/node/pull/23733

Reply:
I’d prefer to undo this unless it’s actually causing trouble on Travis – I don’t think we’ve seen this so far.

Reply:
It makes the tests consistent, but I'll do it in another PR